clang/test/CIR/CodeGen/weak.c

```diff
@@ -0,0 +1,25 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -Wno-unused-value -fclangir-enable -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s -check-prefix=CIR
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -Wno-unused-value -fclangir-enable -emit-llvm %s -o %t.ll
+// RUN: FileCheck --input-file=%t.ll %s -check-prefix=LLVM
+
+//  __attribute__ ((__weakref__("go")))
+extern void B (void);
+static __typeof(B) A __attribute__ ((__weakref__("B")));
+
+void active (void)
+{
+  A();
+}
+
+// CIR:      cir.func extern_weak private @B()
+// CIR:      cir.func @active()
+// CIR-NEXT:   cir.call @B() : () -> () loc(#loc5)
```

Comment:
Remove the `loc(#loc5)` here, those usually make the tests very fragile to unrelated changes.